ui/src/app/components/webcomponents/test-case-steps-list.component.ts

```diff
@@ -191,6 +191,7 @@ export abstract class TestCaseStepsListComponent extends BaseComponent implement
     }
     this.resetAllStepActions();
     testStep.isEditing = false;
+    this.testSteps.content.sort((firstStep,secondStep)=>firstStep.position - secondStep.position);
```

Comment:
why do we need additional sorting in step list? ideally it should be sorted by default, right? @manikanta-reddy-t

Reply:
actually steps are not getting sorted, the steps should be sort based on their position, but it is not being done as expected.
So an additional sort have been added

Reply:
the sptes API supposed to return the sorted by position list, right? @manikanta-reddy-t 
cc: @tarun-testsigma

Reply:
yes that should be the case

Reply:
can you plz check why the steps are not sorted in the API response?